composer.json

```diff
@@ -30,7 +30,9 @@
         "phpstan/phpstan-phpunit": "^0.12",
         "phpstan/phpstan-strict-rules": "^0.12",
         "phpunit/phpunit": "^7.5 || ^8.0 || ^9.0",
-        "gajus/dindent": "^2.0.2"
+        "gajus/dindent": "^2.0.2",
+        "symfony/dom-crawler": "^5.2",
+        "symfony/css-selector": "^5.2"
```

Comment:
I don't add dependencies (even dev dependencies) lightly. However, this made the tests in `BuilderTest::testToctreeGlobOrder` MUCH more precise and readable... and could probably be used in many other places in similar ways.

Reply:
Can we take this as an occasion to set `sort-packages` to true? It will make future diffs easier to understand.